downstream/modules/platform/proc-set-up-virtual-machines.adoc

```diff
@@ -0,0 +1,60 @@
+[id="proc-set-up-virtual-machines"]
+
+= Setting up Virtual Machines for use in an {AutomationMesh}
+
+.Procedure
+
+. SSH into each of the RHEL instances and perform the following steps. 
+Depending on your network access and controls, SSH proxies or other access models might be required. 
++
+Use the following command:
++
+----
+ssh [username]@[host_ip_address]
+----
++
+For example, for an {PlatformNameShort} instance running on {AWS}.
++
+----
+ssh ec2-user@10.0.0.6
+----
+
+. Create or copy an SSH key that can be used to connect from the hop node to the execution node in later steps. 
+This can be a temporary key used just for the automation mesh configuration, or a long-lived key. 
+The SSH user and key are used in later steps.
+
+. Enable RHEL subscription with `baseos` and `appstream` repositories. 
+{PlatformNameShort} RPM repositories are only available through subscription-manager, not the _Red Hat Update Infrastructure_ (RHUI).  
+If you attempt to use any other Linux footprint, including RHEL with RHUI, this causes errors. 
++
+----
+sudo subscription-manager register --auto-attach
+----
++
+If Simple Content Access is enabled for your account, use:
```

Comment:
Maybe link to how you'd find out whether Simple content access is enabled.

Reply:
I've added a link, but apparently 'As of 15-Jul-2022, newly created Red Hat accounts default to having Simple Content Access enabled.'